solutions/devsprint-eduardo-bocato-1/SpaceApp/Resources/Localizable.strings

```diff
@@ -0,0 +1,8 @@
+
+
+// MARK: Common
```

Comment:
How do you think we could go the extra mile and improve this? 🤔

Reply:
Yeap, I got the idea, I'll improve and raise another review. Thank you, mister! 😄